app/auth/signup/email/page.tsx

```diff
@@ -0,0 +1,113 @@
+'use client'
+import { useRef } from 'react'
+import { useForm } from 'react-hook-form'
+import {
+  Box,
+  Flex,
+  FormLabel,
+  FormErrorMessage,
+  FormControl,
+  Input,
+  Heading,
+  useToast
+} from '@chakra-ui/react'
+import { PrimaryButton } from '@/components/button'
+import { signUp } from './action'
+import { useFormResolver, SignUpSchema } from './schema'
+
+export default function SignUp() {
+  const toast = useToast()
+  const isLoading = useRef(false)
+  const toastId = useRef<ReturnType<typeof toast> | undefined>(undefined)
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm<SignUpSchema>({
+    resolver: useFormResolver
+  })
```

Comment:
`errors` does not work. If you try empty email input, input should show error but no errors. You can check `errors` state in console too